modules/react/button/lib/Button.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import {BaseButton, ButtonContainerProps} from './BaseButton';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+/**
+ * Extends all the style properties from Box to our buttons as well as props from ButtonContainerProps.
+ * We omit `ref` since all of our buttons use `createComponent` and already give access to `ref`.
+ * Use this type to extend and customize any one off buttons that you want full control over styling.
+ */
+export interface ButtonProps extends Omit<ButtonContainerProps, 'ref'> {}
+
+export const Button = createComponent('button')({
+  displayName: 'Button',
+  Component: (
+    {
+      children,
+      icon,
+      colors,
+      iconPosition = children ? undefined : 'only',
```

Comment:
what happens with the `start | end` positions?

Reply:
This was just fixed. I fixed it when you called out the actionBar issue